player/vr-360/js/script.js

```diff
@@ -1,29 +1,55 @@
-var conf = {
-  key: '29ba4a30-8b5e-4336-a7dd-c94ff3b25f30',
-  analytics: {
-    key: '45adcf9b-8f7c-4e28-91c5-50ba3d442cd4',
-    videoId: 'vr-360'
-  },
-  style: {
-    aspectratio: '2:1'
-  },
-  playback: {
-    muted: true
-  }
+function isIOS() {
+  return /iPad|iPhone|iPod/.test(navigator.platform);
 };
 
-var source = {
-  dash: 'https://bitmovin-a.akamaihd.net/content/playhouse-vr/mpds/105560.mpd',
-  hls: 'https://bitmovin.com/player-content/playhouse-vr/m3u8s/105560.m3u8',
-  progressive: 'https://bitmovin.com/player-content/playhouse-vr/progressive.mp4',
-  poster: 'https://bitmovin-a.akamaihd.net/content/playhouse-vr/poster.jpg',
-  vr: {
-    startupMode: '2d',
-    startPosition: 180
-  }
-};
+if (!isIOS()) {
```

Comment:
```suggestion
if (isIOS()) {
```

I tested with my iphone and needed to be corrected 👍

Reply:
Ahhh dang, thanks for catching this. :bowing_woman:

Reply:
Fixed in https://github.com/bitmovin/demos/pull/48/commits/5990b84012a43d87d1bc2ff7eaa5a4ac5a35344d